jwst/regtest/test_nircam_image.py

```diff
@@ -22,6 +22,7 @@ def run_detector1pipeline(rtdata_module):
             "--steps.saturation.save_results=True",
             "--steps.superbias.save_results=True",
             "--steps.refpix.save_results=True",
+            "--steps.refpix.use_conv_kernel=False",
```

Comment:
It looks like the only regtest update here is to run NIRCam data with these changes off.  Can we add a regression test that turns them on?